solutions/standard/variables.tf

```diff
@@ -0,0 +1,142 @@
+########################################################################################################################
+# Common variables
+########################################################################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The API Key to use for IBM Cloud."
+  sensitive   = true
+}
+
+variable "existing_resource_group" {
+  type        = bool
+  description = "Whether to use an existing resource group."
+  default     = false
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "The name of a new or an existing resource group in which to provision Secrets Manager resources to."
+}
+
+variable "region" {
+  type        = string
+  description = "The region in which to provision Secrets Manager resources. If using existing Secrets Manager, set this to the region in which it is provisioned in."
```

Comment:
Remove the last sentence. Passing existing SM instance is not a supported use case